src/models/time-sync.ts

```diff
@@ -143,6 +145,14 @@ export class TimeSyncModel implements ModelPluginInterface {
     }, {} as ModelParams);
   }
 
+  private validatePadding(pad: PaddingReceipt): void {
```

Comment:
this method can be rewritten like this to be more organized, with doc

```
  /**
   * Checks if `error on padding` is enabled and and padding is needed. If so, then throws error.
   */
  private validatePadding(pad: PaddingReceipt): void {
    const {start, end} = pad;
    const isPaddingNeeded = start || end;

    if (this.errorOnPadding && isPaddingNeeded) {
      const paddingDescription =
        start && end ? 'start and end' : start ? 'start' : 'end'; // this padding description thing can be incapsulated in AVOIDING_PADDING logic. BTW function strings have unit tests

      throw new InputValidationError(AVOIDING_PADDING(paddingDescription));
    }
  }
```

Reply:
i like it, much cleaner code. Thanks :) also added tests for string functions. 
notice i put 2 different string functions since there is 1 where description is free text and the other is determined based on flags (start, end)